src/Frost.ts

```diff
@@ -341,4 +344,24 @@ export class Frost {
         throw e
       })
   }
+
+  getPublicKeys(cuid?: string): Promise<{ readonly publicKey: string }> {
```

Comment:
Maybe name this PR as to refer to this new endpoint being added as the main thing rather than adding cuid?